x-pack/plugins/security_solution/common/cti/constants.ts

```diff
@@ -63,3 +63,5 @@ export const DEFAULT_EVENT_ENRICHMENT_TO = 'now';
 
 export const TI_INTEGRATION_PREFIX = 'ti';
 export const OTHER_TI_DATASET_KEY = '_others_ti_';
+
+export const THREAT_PIT_KEEP_ALIVE = '1h';
```

Comment:
```
The value (e.g. 1m, see [Time units](https://www.elastic.co/guide/en/elasticsearch/reference/current/api-conventions.html#time-units)) does not need to be long enough to process all data — it just needs to be long enough for the next request.
```

From docs here: https://www.elastic.co/guide/en/elasticsearch/reference/current/point-in-time-api.html

I don't think you want this to be 1 hour but rather a lower time. Just enough that you can re-call and process things again. So for example if the elastic search will be-recalled for sure within 5 minutes, 5 minutes is a better default.